composer.json

```diff
@@ -20,7 +20,9 @@
         "symfony/security-core": "^3.4|^4.4|^5.0"
     },
     "require-dev": {
+        "symfony/cache": "^3.4|^4.4|^5.0",
         "symfony/phpunit-bridge": "^5.2",
+        "doctrine/cache": "^1.6",
```

Comment:
`doctrine/cache` was being installed as a transient dependency, so I've added it (with the same minimum as what the Symfony cache component currently tests against) to be explicitly loaded for dev.